source/data_analysis_SPRING2023.Rmd

```diff
@@ -1,229 +1,1296 @@
+# enkel nodig indien je interactief werkt en de code van deze chunks nodig hebt
+# clipr::write_clip(rmd)
+
+knit(text = rmd, quiet = TRUE) %>%
+  cat()
+```
+
+
+# Data modellering
+
+## Soortenrijkdom
+
+We fitten de modellen voor beide soortengroepen samengenomen in hetzelfde model, maar we laten elke covariaat in interactie gaan met de soortengroep zodat we aparte parameterschattingen per soortengroep bekomen.
+
+We maken geen correctie voor het aantal gevangen individuen (door dit toe te voegen als covariaat): omwille van het hoge aantal gevallen waar er 0 soorten zijn (en dus 0 individuen) geeft deze correctie problemen in het model (0 individuen is een perfecte, maar ook onzinnige, voorspeller voor 0 soorten).
+
+Gegevens van de tijdsreekens en de 11de pantrap op locatie 1 werden niet meegenomen in de modellen.
+
 ```{r}
-submet <- unique(apidae %>% 
-                   filter(method_cd == "TS") %>% 
-                   pull(SPRING_code))
+data_sp_rich <- apoidea %>%
+  mutate(taxgroup = "Apoidea") %>%
+  bind_rows(
+    syrphidae %>%
+      mutate(taxgroup = "Syrphidae")
+  ) %>%
+  filter(time_series == 0) %>%
+  mutate(
+    maand = as.factor(month(date_b))) %>%
+  group_by(
+    sample_code, location_code, method_combi, 
+    method_cd, spring_code, uv, level, maand, taxgroup) %>%
+  summarise(
+    n_species = n_distinct(species_nm, na.rm = TRUE),
+    n_ind = sum(no_ind),
+    .groups = "drop"
+  )
+```
 
-x <- list()
+### SPRING protocol
 
-for (i in submet) {
-  int <- unique(apidae %>% 
-                  filter(`time series?` == 0,
-                         method_cd == "TS",
-                         SPRING_code == i) %>% 
-                  select(`species_nm author + year`))
-  colnames(int) <- c(i)
-  x <- append(x, int)
-}
+Het eerste model heeft enkel betrekking op de SPRING methode sensu stricto. 
 
-ggvenn(x)
+```{r}
+data_sp_rich_spring <- data_sp_rich %>%
+  filter(spring_code == "s.s.")
 ```
 
+
+```{r, fig.cap = "Aantal keren dat n aantal soorten in één pan trap of subtransect van 50m gevonden werd."}
+ggplot(data_sp_rich_spring, aes(x = n_species)) +
+  geom_histogram(
+    aes(fill = taxgroup),
+    position = position_dodge())
+```
+
+
+```{r, fig.cap = "Aantal individuen in één pan trap of subtransect van 50m ten opzichte van het aantal soorten in die pan trap of dat subtransect."}
+ggplot(data_sp_rich_spring) +
+  geom_point(aes(x = n_ind, y = n_species)) +
+  facet_wrap(~ taxgroup, scales = "free")
+```
+
+```{r}
+model0 <- glmmTMB(
+  n_species ~ 
+    (location_code
+  + maand
+  + method_cd) * taxgroup
+  ,
+  family = "poisson",
+  ziformula = ~ taxgroup,
+  na.action = na.fail,
+  data = data_sp_rich_spring
+)
+```
+
+```{r check1, fig.height=12, fig.cap = "Visuele controle van verschillende modelaannames."}
+performance::check_model(model0)
+```
+
+```{r, fig.cap = "Verwacht aantal soorten per maand, soortgroep en SPRING methode op basis van ons model."}
```

Comment:
```suggestion
```{r, fig.cap = "Verwacht aantal soorten in een pantrap-set of transectsegment van 50 m per maand, soortgroep en SPRING methode op basis van ons model."}
```

Kan je dit ook zo aanpassen in de andere figuren?